src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -519,6 +519,46 @@ static function debug_data() {
 			'value' => ( is_array( $imagick_version ) ? $imagick_version['versionString'] : $imagick_version ),
 		);
 
+		if ( ! function_exists( 'ini_get' ) ) {
+			$info['wp-media']['fields']['ini_get'] = array(
+				'label' => __( 'File upload settings' ),
+				'value' => sprintf(
+					/* translators: %s: ini_get() */
+					__( 'Unable to determine some settings, as the %s function has been disabled.' ),
+					'ini_get()'
+				),
+				'debug' => 'ini_get() is disabled',
+			);
+		} else {
+			// Get the PHP ini directive values.
+			$post_max_size    = ini_get( 'post_max_size' );
+			$upload_max_size  = ini_get( 'upload_max_filesize' );
+			$max_file_uploads = ini_get( 'max_file_uploads' );
+			$effective        = min( $post_max_size, $upload_max_size );
```

Comment:
I'm not quite sure that this will work as intended. For example, `min('4M', '500K')` returns `4M`, and not `500K` as expected. I think these values should go through `parse_ini_size()` method before comparison.

Reply:
Thanks for the catch! This should be resolved in c1cddd1.

Reply:
Awesome, thank you.
// end suggestion :)